src/atoms.ts

```diff
@@ -246,6 +246,35 @@ function parseAtom(
             });
         }
 
+        case "quiz": {
+            const atom = atoms.quizzes?.find(quiz => quiz.id === id);
+            if (atom?.data?.kind !== "quiz" || !id) {
+                return err(`No atom matched this id: ${id}`);
+            }
+
+            const { content } = atom?.data?.quiz;
+
+            if (!content) {
+                return err(`No content for atom: ${id}`);
+            }
+
+            const questions = content.questions.map(question => {
+                return {
+                    text: question.questionText,
+                    ...question,
+                    answers: question.answers.map(answer => {
+                        return {
+                            text: answer.answerText,
+                            isCorrect: !!answer.weight,
```

Comment:
`isCorrect` is derived from weighting?

Reply:
I think so, it seems to check out on the quiz I tried 🤷‍♂️ It sounds like some answers are worth more than others